### ISSUE TITLE
bundler not using selected compile options

### ISSUE THREAD

**Issue by dkondrad** [sveltejs/svelte-repl#79]
**Description**

The bundler doesn't pass any of the selected compile options when checking for errors or warnings.

This causes spurious messages when custom elements are used, such as in this very simple repl:
https://svelte.dev/repl/85f4c30c8f8440e58a96eccfdf8f64b1?version=3.16.5

You'll notice the code is generated correctly, but the REPL still complains due to the additional compile step [here](https://github.com/sveltejs/svelte-repl/blob/master/src/workers/bundler/index.js#L184).

Not sure why we're running compilation twice anyway... seems that the compiler service worker should pass back the results so that we can display them.

** Priority **
Low, but it will drive users crazy trying to figure out what they did wrong.

**Comment by dkondrad** [comment]
Maybe the real issue is that the user files aren't even meant to be picked up by this worker?

**Comment by claudiu04** [comment]
My original post: https://github.com/sveltejs/svelte/issues/4132#issue-540406214
**REPL example**: https://svelte.dev/repl/489ee8acd10848b0bb1feb2535bd6cc5?version=3.16.5
created locally:
1. When I registering `[ customElement: true ] `in the config i get
<img width="1428" alt="Screen Shot 2562-12-21 at 18 15 17" src="https://user-images.githubusercontent.com/12200607/71307314-1a2d8b80-241f-11ea-9c64-3e1ded81f576.png">

2. If I do not register my element in the config I do not get any error, but neither my element is registered :(
<img width="1433" alt="Screen Shot 2562-12-21 at 18 15 03" src="https://user-images.githubusercontent.com/12200607/71307315-1ef23f80-241f-11ea-941c-3e0d88f9b0ca.png">

I will like to achieve the behaviour mentioned in the **first example** when I register the component in the config.

Can some help with this ?

Thank you in advance !

**Comment by dkondrad** [comment]
@claudiu04, this bug is for the REPL itself... please don't spam this issue with cross-posts.

Secondly, all components, even the top level "App" one must have a tag option when compiling with `customElement: true`, whether `{null}` or otherwise.

**Comment by claudiu04** [comment]
I agree. As soon as i set the `customElement: true` tag in the config, the app.mjs its will through an error. Even if I do not register any component. 

Furthermore, when trying to debug app.mjs the SvelteElement component it seems that the compiler will through an Illegal constructor error,  but it will not be triggered until you will call the component or register via customElement: true.

**Comment by dkondrad** [comment]
@Conduitry, ignore the noise from that other person, this is a bug... see my original post.
